src/components/DropdownElement/DropdownElement.tsx

```diff
@@ -0,0 +1,61 @@
+import colors from '../../styles/colors';
+import { DropdownElementProps } from './types';
+import { DropdownElementStyles } from './DropdownElement.styles';
+import { Icon } from '../Icon';
+import { iconTypes } from '../Icon/collection';
+import React from 'react';
+
+const { DivContainerStyled, DivStyled, DivImageStyled, DivTextStyled } =
+    DropdownElementStyles;
+
+const DropdownElement: React.FC<DropdownElementProps> = ({
+    backgroundColor = colors.blueDark,
+    height,
+    icon = iconTypes.check,
+    iconColor = 'white',
+    iconSize = 20,
+    id = String(Date.now()),
```

Comment:
this ID thing was my idea, it worked in test but if there element renders super fast it can cause warnings, lets remove this please

Reply:
removed. Maybe we could use `uuid` for all the Components (I think that's the standard way to go for react apps)